status-im/status-legacy
# Add support for SAI/DAI

Fixes #9491 

### Summary

Adds support for Multicollateral DAI and renames single collateral DAI to SAI

### Review notes
Make sure to look at the new DAI icons and make sure they look right.  I used GIMP to resize the provided DAI badge to the right pixel sizes but want to make sure there aren't any distortions.

#### Platforms
- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

1. Open Status
2. Go to Wallet screen
3. Tap "..." in upper right corner and select Manage Assets from the Bottom Sheet
4.  Add DAI and SAI to Assets and verify that they appear in the Wallet

status: ready 

![Screenshot_20191119-210157](https://user-images.githubusercontent.com/17355484/69202750-87908880-0b10-11ea-83b3-5fd96067a59d.jpg)
